config.codekit3

```diff
@@ -552,6 +549,32 @@
       "q" : 100,
       "rq" : 75
     },
+    "\/src\/assets\/images\/optin-themes\/blogger\/blogger-lightbox.png" : {
+      "ft" : 32768,
+      "iS" : 23014,
+      "oA" : 0,
+      "oAP" : "\/src\/assets\/images\/optin-themes\/blogger\/blogger-lightbox.png",
```

Comment:
Ensure you exclude your codekit config file from every PR @ShoboySnr

Reply:
Okay will do that